lib/msal-common/src/error/NativeAuthError.ts

```diff
@@ -3,11 +3,21 @@
  * Licensed under the MIT License.
  */
 
-import { AuthError } from "@azure/msal-common/node";
-import { StringUtils } from "../utils/StringUtils.js";
+import { NativeBrokerStringUtils } from "../utils/NativeBrokerStringUtils.js";
+import { AuthError } from "./AuthError.js";
 
+/**
+ * Error class for MSAL Runtime errors that preserves detailed broker information
+ */
 export class NativeAuthError extends AuthError {
```

Comment:
Let's rename this to `PlatformBrokerError` as we've had a bit of a naming conflict with the CIAM team for `Native Auth`